tests/templates

```diff
@@ -0,0 +1 @@
+../../../templates
```

Comment:
This should be in `tests/roles/linux-system-roles.nbde_server/` instead of `tests/`

Reply:
fixed